### DISCUSSION TITLE
Energy price history

### DISCUSSION THREAD

**Discussion by Jellybeanz1980** [UtilitechAS/amsreader-firmware#602] [Ideas]
Hello,

A graph that shows the price history for the last 24 hours, that is synchronized with the "Energy use last 24 hours (kWh)" graph would be very nice. With this it would be possible to see how much energy is used during which price period. 

Best regards
Flemming

**Comment by ArnieO** [comment]
It would become a bit more complex to read the graph, but I agree it would be a good idea. Could for instance be a line graph overlay with a secondary axis for the price scale. 
Would something like that be feasible in the graphing tool, @gskjold?

**Reply by gskjold** [reply]
It might be, I'll make a note of it for v2.3 development
